website/static/js/fileRevisions.js

```diff
@@ -91,6 +103,17 @@ var RevisionsViewModel = function(node, file, editable) {
 
 RevisionsViewModel.prototype.fetch = function() {
     var self = this;
+
+    // Dataverse Hack: Only latest version is editable;
+    // Users without edit permission should not see revision table
+    if (self.file.provider === 'dataverse') {
+        if (self.editable()) {
+            self.editable(urlParams.version === 'latest');
+        } else {
+            return [];
```

Comment:
This return value is a little misleading. Currently, `fetch` doesn't return anything. In the future, it should be refactored to return a promise that resolves to list of `Revisions`. I think for now, you can just return nothing here.